bento-platform/bentoV2
# Qa/v2.6.5

This PR fixes some issues related to paths within the bento_drs container: in the bento_drs project there is still a directory named "chord_drs" that contains the application code. Due to recent changes in branch qa/v2.6.2 some references to this internal directory had been replaced by "bento_drs" which does not apply in this case.

Also, this PR improves using the bento_drs container in dev mode, by mounting the local git repository, similarly as what is done with other services such as Katsu and bento_wes.

How to test:
- remove the bento_drs container and the bento_drs image on your local installation
- make sure that the commands `make run-drs` and `make run-drs-dev` are working as expected (an image is built, a container is spun and remains active). Note that the container and its image must be cleared between each test to avoid reusing the same image/container
- in dev mode, make sure that local changes within the bentov2/lib/drs/bento_drs folder are visible within the dev container (e.g. same branch)
